Cookbook/Technical-Documents/WhatIDontKnow.md

```diff
@@ -0,0 +1,183 @@
+
+### The iOS team
+One of the greatest sources of our collective knowledge is a #ios-questions slack channel. You can ask there any question and the whole team will try to help you.
+
+## In the tower of Babel
+When working in such a vibrant and dynamic environment it is very easy to lose a track of things that should happen. This is a shortlist of topics that should help you to get through the onboarding time.
+
+### Communication and Collaboration
+Effective and open communication is crucial in a company like Babylon. Without it, everything falls apart. If there is one thing you should remember from this guide let me suggest this:
+Context is king!
+
+When you’re communicating digitally, you never quite know what the other person is doing at that moment. They might be at their desk just like you are, or they may be frantically rushing to a meeting, only responding “Yes” to your question and not elaborating because they don’t have time.
+
+Without understanding the other person’s context, you might think that person doesn’t care about the issue you brought up when they’re really just running to a meeting room. With the information on that person’s context, all of a sudden, the curt answers make sense: It’s not that your coworker doesn’t care, they are just indisposed at the moment.
+
+Prefacing communication with your context can help to prevent any miscommunication when things are out of the ordinary. Let team members know when you are heads down on a project and can’t respond to questions right away.
+
+Over communicating is always better than making assumptions.
+
+To help you with a difficult art of digital communication, here are some resources for you:
+
+[Remote team communication](https://zapier.com/learn/remote-work/remote-team-communication/)
+
+Why you should read it: This will give you a bunch of tips on how to communicate better digitally.
+
+[Remote communication](https://about.gitlab.com/2016/03/23/remote-communication/index.html)
+
+Why you should read it: It has some universal commandments about communicating asynchronously (between humans ;) )
+
+[Communication: how to be a better software developer](https://medium.com/techspiration-ideas-making-it-happen/communication-how-to-be-a-better-software-developer-869c50767701)
+
+Why you should read it: A bit longer read, but presents good practices we all should strive to achieve.
+
+### Am I missing something?
+This is our FAQ section. Every question that you may have have a short answer and/or links to additional resources. If during your onboarding you ever find yourself asking a question that is not on this list, create a new item, find an answer with your buddy and update the board.
+
+#### List of events you should have in your calendar
+- Your squad’s daily/retro/ planning
+- Babylon standup
+- Frontend demo
+- Proposal meetings
+- PR party
+- Lunch and Learn
+- iOS Banter
+- 1-on-1 with your manager
```

Comment:
What do you think about adding a small description for  the meetings?
I was a bit lost what was happening, especially when I joined my first banter or got invited to the PR party.

Reply:
That's a great idea. Will add that

Reply:
Had some time today :) I was thinking something like this, but feel free to modify. I haven't added any formatting either, maybe the explanation could be italic for better readability?

- Your squad’s daily/retro/planning: ask your product or delivery manager how your squad organises these and make sure they invited you to all of them.
- Babylon standup: this weekly standup helps us keep up with all things happening in the different teams in Babylon.
- Frontend demo: curious what's going on in other Frontend teams? Participate at Frontend demo and have a glimpse of their work.
- Proposal meetings: Have a suggestion about how we can improve our way of work? Create a proposal for it so we can exchange ideas about it. You can find our proposals in the Cookbook.
- PR party: Seen your name group up with your teammates and a bunch of PR numbers in the ios-underground channel? Congratulations you got invited to your first PR party!
The first person on the list will slack you and invite you to a Zoom call where you will review all the assigned pull requests.
- Lunch and Learn: grab your lunch and join the team to watch a video or listen to someone's presentation.
- iOS Banter: we gather together every week to share our concerns and successes.
- 1-on-1 with your manager: biweekly catch up with your manager where you can discuss anything, they are to help you grow to be the best you can be.

Reply:
You are missing the iOS Retro meeting.
Maybe you could link these to the document that describes the chapter meetings https://github.com/babylonhealth/ios-playbook/blob/master/Cookbook/Technical-Documents/meetings-purpose.md